iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.7.0</Version>
+    <Version>1.7.1</Version>
```

Comment:
What changes do we have to IoTHub DeviceClient?

Reply:
The following change was made to the ProductInfo class: Update ProductInfo to use semver-compliant AssemblyInformationalVersion